src/containers/admin/EditEventForm.jsx

```diff
@@ -1,9 +1,30 @@
+import React from 'react';
 import { connect } from 'react-redux';
 import { withFormik } from 'formik';
 
+import { useHistory } from 'react-router-dom';
 import EditEventform from '../../components/EditEventForm';
-import { getMonthIndex } from '../../utils';
-import { editEvent } from '../../actions/adminActions';
+import { getMonthIndex, notify } from '../../utils';
+import { editEvent, deleteEvent } from '../../actions/adminActions';
+
+function EditEvntFormWithDelete(props) {
+  const history = useHistory();
+  function handleDelete() {
+    props
+      .deleteEvent(props.values.uuid)
+      .then((resp) => {
+        notify(resp);
```

Comment:
There's no need to notify with the response since you already call notify in the action.